dataactvalidator/config/sqlrules/a18_cross_file.sql

```diff
@@ -1,10 +1,26 @@
 SELECT
-    approp.row_number,
-    approp.tas,
-    approp.gross_outlay_amount_by_tas_cpe,
-    SUM(op.gross_outlay_amount_by_pro_cpe) as gross_outlay_amount_by_pro_cpe_sum
-FROM appropriation approp JOIN object_class_program_activity op
-ON approp.tas = op.tas AND approp.submission_id = op.submission_id
+	approp.row_number,
```

Comment:
Confirmed with Kristine this morning that the original implementation of A18 should stand, instead of the updated one with additional math that we received recently. The text will be updated in the next RSS, so we will be leaving A18 as-is for now.